### PR TITLE
Reconfigure comments xpromo for non-experiment feature flag

### PR DESCRIPTION
**Pull request by schwers**

### CODE REVIEW

File: src/lib/eventUtils.js [hunk 1]
```diff
@@ -180,17 +180,18 @@ export function trackPagesXPromoEvents(state, additionalEventData) {
     const ineligibilityReason = shouldNotShowBanner();
     if (ineligibilityReason) {
       trackXPromoIneligibleEvent(state, additionalEventData, ineligibilityReason);
-    } else if (xpromoIsEnabledOnPage(state) && xpromoIsEnabledOnDevice(state)) {
+    } else if (xpromoIsEnabledOnDevice(state)) {
```
Comment (schwers) [inline src/lib/eventUtils.js]: Note: `xpromoIsEnabledOnPage` just checks `isEligibleListingPage || isEligibleCommentsPage`, so we don't need to duplicate that here

**Comment by schwers** [comment]
👓  @tomsoir && @prashtx

**Comment by tomsoir** [comment]
:fish:

**Comment by schwers** [comment]
💇

**Comment by prashtx** [comment]
:fish:

**Comment by tomsoir** [comment]
🐟

**Pull request merged**